package.json

```diff
@@ -22,6 +22,7 @@
     "merge-stream": "0.1.6",
     "rimraf": "2.2.8",
     "run-sequence": "1.0.1",
+    "stylus": "~0",
```

Comment:
Yeah, this seems to break the settings gear. I might just duplicate the lines that need the updated version.